src/tapis-ui/components/jobs/JobLauncher/FieldArray.tsx

```diff
@@ -44,7 +46,12 @@ export function FieldArray<T>({
 
   return (
     <div className={styles.array}>
-      <Collapse title={title} note={`${fields.length} items`}>
+      <Collapse
+        open={required.length > 0}
+        title={title}
+        note={`${fields.length} items`}
+        isCollapsable={isCollapsable && required.length > 0}
```

Comment:
I think the client component should completely control isCollapsable

```
isCollapsable={isCollapsable}
```

Then the client can do:

```
<FieldArray isCollapsable={required.length === 0} />
```

Leave it up to the client to decide? Either that or still have this be:

```
{isCollapsable && required.length === 0}
```

Reply:
Agreed, client should have control. isCollapsable={isCollapsable} will be implemented